### ISSUE TITLE
GCS Storage driver not registered in registry v2.7.0

### ISSUE THREAD

**Issue by ghost** [docker/distribution#2819]
Hi everyone,

We deployed recently new registries and realised that the registry container is constantly crashing due to:

```
panic: StorageDriver not registered: gcs
goroutine 1 [running]:
github.com/docker/distribution/registry/handlers.NewApp(0xefe940, 0xc0003b1f50, 0xc0003a1c00, 0x0)
        /go/src/github.com/docker/distribution/registry/handlers/app.go:126 +0x2fbb
github.com/docker/distribution/registry.NewRegistry(0xefe940, 0xc0003b1f50, 0xc0003a1c00, 0xe, 0x0, 0x0)
        /go/src/github.com/docker/distribution/registry/registry.go:105 +0x1dc
github.com/docker/distribution/registry.glob..func1(0x15972c0, 0xc0003c03d0, 0x1, 0x1)
        /go/src/github.com/docker/distribution/registry/registry.go:63 +0x187
github.com/docker/distribution/vendor/github.com/spf13/cobra.(*Command).execute(0x15972c0, 0xc0003c0390, 0x1, 0x1, 0x15972c0, 0xc0003c0390)
        /go/src/github.com/docker/distribution/vendor/github.com/spf13/cobra/command.go:495 +0x191
github.com/docker/distribution/vendor/github.com/spf13/cobra.(*Command).Execute(0x1597460, 0xc0001f5f88, 0xc00007c058)
        /go/src/github.com/docker/distribution/vendor/github.com/spf13/cobra/command.go:560 +0x2f4
main.main()
        /go/src/github.com/docker/distribution/cmd/registry/main.go:23 +0x2d
```

The command used to run the registry:

```
/usr/bin/docker run --name registry -p 5000:5000 -e REGISTRY_STORAGE=gcs -e REGISTRY_STORAGE_GCS_BUCKET=my-fancy-bucket -e REGISTRY_STORAGE_GCS_KEYFILE=/var/lib/registry/gcs/docker-registry.json -v /etc/gce/service-accounts/:/var/lib/registry/gcs registry:2.7.0
```

We rolled back to 2.6.2 and everything seems to work fine. I tried different storage like s3 and get a proper error about not setting the required env variable. Only gcs crashes at startup with the error above.

Let me know if I can help in any ways.

**Comment by caervs** [comment]
Ah yup.

@dmcgowan looks like we made the [env var in the Dockerfie](https://github.com/docker/distribution/blob/master/Dockerfile#L4) is `DOCKER_BUILDTAGS`, but it's `BUILDTAGS` in the [Makefile](https://github.com/docker/distribution/blob/master/Makefile#L32). Simple enough fix. I'll get a PR out. I think we should shoot to have this in `2.7.1`.

**Comment by thaJeztah** [comment]
Looks like this was included in `v2.7.1`: https://github.com/docker/distribution/blob/v2.7.1/Dockerfile#L4, but didn't close the ticket

So this can be removed from the v2.7.2 milestone and closed

**Comment by thaJeztah** [comment]
Looks like https://github.com/docker/distribution/pull/2821 was made directly against the `release/2.7` branch, and didn't get into master; opened https://github.com/docker/distribution/pull/3086 to get it into master
